CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 9/14/17
+
+##### `resourceful-redux@1.1.0-beta`
+
+**New Features**
+
+- The `resourceful-redux` library now supports lists, which is a necessary step
+  toward our 2.0.0 milestones in which we track labels & lists separately for
+  different CRUD use cases. For more info, refer to #187 or read further in
+  our [docs](https://resourceful-redux.js.org/).
```

Comment:
I'd deemphasize the 2.0.0 milestone here, and maybe elaborate more on:

> in which we track labels & lists separately for different CRUD use cases

Right now, this is pretty vague. Why would anyone care about lists? What are they for?

Lists are a replacement for labels for tracking ordered collections of resources. I think we should state that clearly before mentioning anything about versions.